PowerShellEditorServices.build.ps1

```diff
@@ -42,7 +42,7 @@ $script:dotnetTestArgs = @("test") + $script:dotnetBuildArgs + $TestArgs + @(
 $script:IsNix = $IsLinux -or $IsMacOS
 # For Apple M1, pwsh might be getting emulated, in which case we need to check
 # for the proc_translated flag, otherwise we can check the architecture.
-$script:IsAppleM1 = $IsMacOS -and ((sysctl -n sysctl.proc_translated) -eq 1 -or (uname -m) -eq "arm64")
+$script:IsAppleM1 = $IsMacOS -and ((sysctl -n sysctl.proc_translated 2> $null) -eq 1 -or (uname -m) -eq "arm64")
```

Comment:
May be better to use `[system.runtime.InteropServices.RuntimeInformation]::ProcessArchitecture`

Reply:
Thanks!